policy:
  type: "random"
  min_vectors: 1
  max_vectors: 3